scripts/nayduck.py

```diff
@@ -203,6 +207,24 @@ def main():
         'tests': list(tests)
     }
 
+    if args.run_locally:
+        for test in post["tests"]:
+            # TODO: use parses similar to nayduck, where is it?
```

Comment:
https://github.com/near/nayduck/blob/master/lib/testspec.py
This is not the first time I wish there was a way to share code between this repo and nayduck.